### ISSUE TITLE
Move our work so far to bare react native

### ISSUE THREAD

**Issue by mylesieong** [mylesieong/Seated-Exercises-in-Vehicles#168]

**Comment by mylesieong** [comment]
Its not needed anymore.

Regarding the triggered build job. I cannot find any work around to take advantages on both Expo's accessibility for development and Bare RN's flexibility for deployment, so we will go for Expo since its more relevant to us at the moment.
However I kept the triggered build job here to run npm audit for us which at least could make sure we don't have regression on fixed vulnerabilities. Please check when you have time.
